src/ResourceManager/Profile/Commands.Profile/AzureRmAlias/Mappings.cs

```diff
@@ -33,2377 +34,12 @@ public class Mappings
-    'Update-AzSqlSyncMember': 'Update-AzureRmSqlSyncMember',
-    'Enable-AzSqlServerAdvancedThreatProtection': 'Enable-AzureRmSqlServerAdvancedThreatProtection',
-    'Set-AzSqlDatabaseLongTermRetentionPolicy': 'Set-AzureRmSqlDatabaseLongTermRetentionPolicy',
-    'Remove-AzSqlDatabaseRestorePoint': 'Remove-AzureRmSqlDatabaseRestorePoint',
-    'New-AzSqlSyncMember': 'New-AzureRmSqlSyncMember',
-    'Remove-AzSqlServerCommunicationLink': 'Remove-AzureRmSqlServerCommunicationLink',
-    'Get-AzSqlServerAuditingPolicy': 'Get-AzureRmSqlServerAuditingPolicy',
-    'Add-AzSqlServerKeyVaultKey': 'Add-AzureRmSqlServerKeyVaultKey',
-    'Set-AzSqlDatabaseServerAuditingPolicy': 'Set-AzureRmSqlDatabaseServerAuditingPolicy',
-    'Update-AzSqlDatabaseVulnerabilityAssessmentSettings': 'Update-AzureRmSqlDatabaseVulnerabilityAssessmentSettings',
-    'Stop-AzSqlElasticPoolActivity': 'Stop-AzureRmSqlElasticPoolActivity',
-    'Stop-AzSqlServerUpgrade': 'Stop-AzureRmSqlServerUpgrade',
-    'New-AzSqlDatabaseFailoverGroup': 'New-AzureRmSqlDatabaseFailoverGroup',
-    'Remove-AzSqlDatabaseFailoverGroup': 'Remove-AzureRmSqlDatabaseFailoverGroup',
-    'Get-AzSqlDatabaseGeoBackupPolicy': 'Get-AzureRmSqlDatabaseGeoBackupPolicy',
-    'Get-AzSqlElasticPoolRecommendedAction': 'Get-AzureRmSqlElasticPoolRecommendedAction',
-    'Get-AzSqlDatabaseAuditing': 'Get-AzureRmSqlDatabaseAuditing',
-    'Set-AzSqlElasticPoolAdvisorAutoExecuteStatus': 'Set-AzureRmSqlElasticPoolAdvisorAutoExecuteStatus',
-    'New-AzSqlSyncGroup': 'New-AzureRmSqlSyncGroup',
-    'Get-AzSqlServerThreatDetectionPolicy': 'Get-AzureRmSqlServerThreatDetectionPolicy',
-    'Get-AzSqlServerActiveDirectoryAdministrator': 'Get-AzureRmSqlServerActiveDirectoryAdministrator',
-    'Get-AzSqlServerAdvisor': 'Get-AzureRmSqlServerAdvisor',
-    'Set-AzSqlDatabaseAdvisorAutoExecuteStatus': 'Set-AzureRmSqlDatabaseAdvisorAutoExecuteStatus',
-    'New-AzSqlServerVirtualNetworkRule': 'New-AzureRmSqlServerVirtualNetworkRule',
-    'Stop-AzSqlDatabaseExecuteIndexRecommendation': 'Stop-AzureRmSqlDatabaseExecuteIndexRecommendation',
-    'Remove-AzSqlServerDisasterRecoveryConfiguration': 'Remove-AzureRmSqlServerDisasterRecoveryConfiguration',
-    'Remove-AzSqlSyncAgent': 'Remove-AzureRmSqlSyncAgent',
-    'Get-AzSqlServerUpgradeHint': 'Get-AzureRmSqlServerUpgradeHint',
-    'Get-AzSqlDatabaseImportExportStatus': 'Get-AzureRmSqlDatabaseImportExportStatus',
-    'Add-AzSqlServerTransparentDataEncryptionCertificate': 'Add-AzureRmSqlServerTransparentDataEncryptionCertificate',
-    'Get-AzSqlDatabaseRecommendedAction': 'Get-AzureRmSqlDatabaseRecommendedAction',
-    'Convert-AzSqlDatabaseVulnerabilityAssessmentScan': 'Convert-AzureRmSqlDatabaseVulnerabilityAssessmentScan',
-    'Resume-AzSqlDatabase': 'Resume-AzureRmSqlDatabase',
-    'Set-AzSqlDatabaseAuditingPolicy': 'Set-AzureRmSqlDatabaseAuditingPolicy',
-    'Get-AzSqlServerBackupLongTermRetentionVault': 'Ge
```

Comment:
This isn't necessary. In the `DeserializeObject` call, just do `File.ReadAllText("Mappings.json")`.